client/my-sites/plans/plan-overview/plan-feature/index.jsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import CompactCard from 'components/card/compact';
+
+const PlanFeature = ( { button, description, heading } ) => {
+	return (
+		<CompactCard className="plan-feature">
+			<div className="plan-feature__description">
+				<strong>{ heading }</strong>
+				<em>{ description }</em>
+			</div>
+			{ button &&
+				<Button
```

Comment:
Could we use `compact` buttons here? I think it will look better given the space and the repetition.